src/02-common-principles.md

```diff
@@ -377,7 +377,9 @@ data is expressed in SI units or SI derived units, the units MAY be specified in
 the sidecar JSON file. In case non-standard prefixes are added to SI or non-SI
 units, these non-standard prefixed units MUST be specified in the JSON file.
 See [Appendix V](99-appendices/05-units.md) for a list of standard units and
-prefixes.
+prefixes. Note also that for the *formatting* of SI units, the
+[CMIXF-12](https://people.csail.mit.edu/jaffer/MIXF/CMIXF-12) convention for
+encoding units is RECOMMENDED.
```

Comment:
```suggestion
encoding units is REQUIRED.
```

Reply:
can we add an example here? What would be CMIXF-12 compatible and what would not be? Just to make sure that users don't get lost in trying to understand another standard ...

Reply:
Good point, ... there are some examples in the appendix, but I'll also add some here. :+1: